src/modules/fancyzones/lib/Settings.h

```diff
@@ -34,6 +34,8 @@ struct Settings
     bool appLastZone_moveWindows = false;
     bool openWindowOnActiveMonitor = false;
     bool restoreSize = false;
+    bool quickLayoutSwitch = false;
+    bool flashZonesOnQuickSwitch = false;
```

Comment:
```suggestion
    bool quickLayoutSwitch = true;
    bool flashZonesOnQuickSwitch = true;
```
Let's have these options on by default.